app/profile/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { createClientSsr } from "@/utils/supabase/client";
+import UserProfile from "@/components/profile/UserProfile";
+import {GetUser} from "@/utils/userData";
```

Comment:
we'll have to do passes with `prettier` every now and then, but i don't care enough to let it bottleneck dev

Reply:
https://github.com/quinn-dougherty/nextjs-with-supabase/commit/1fe08c1fdbaba61531fc960f8fb885b31c604f43